tendrl/node_agent/manager/manager.py

```diff
@@ -82,6 +84,18 @@ def _run(self):
         LOG.info("%s complete" % self.__class__.__name__)
 
 
+def update_node_context(manager, machine_id, tags=""):
```

Comment:
This should be in sync with https://github.com/Tendrl/node-agent/pull/134
one update should not overwrite the other